nntrainer/tensor/cache_loader.cpp

```diff
@@ -28,13 +28,16 @@ namespace nntrainer {
 CacheLoader::CacheLoader(std::shared_ptr<CachePool> cache_pool) :
   pool(cache_pool),
   load_task_executor(nullptr),
-  unload_task_executor(nullptr) {}
+  unload_task_executor(nullptr),
+  thread_pool_(nullptr) {}
```

Comment:
I think it is more like replace the load_task_executor and unload_task_executor with now executor.